modules/nf-core/cnvkit/genemetrics/main.nf

```diff
@@ -0,0 +1,40 @@
+
+process CNVKIT_GENEMETRICS {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::cnvkit=0.9.9" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/cnvkit:0.9.9--pyhdfd78af_0':
+        'quay.io/biocontainers/cnvkit:0.9.9--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(cnr), path(cns)
+
+    output:
+    tuple val(meta), path("*.tsv"), emit: tsv
+    //tuple val(meta), path("*.cnn"), emit: cnn
```

Comment:
why commented?

Reply:
Sorry, my bad, it is not an output file from this process. I removed it ☺️